_scripts/gen-faq-toc.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"regexp"
+	"strings"
+)
+
+func must(err error, fmtstr string, args ...interface{}) {
+	if err != nil {
+		log.Fatalf(fmtstr, args...)
+	}
+}
+
+func usage() {
+	fmt.Fprintf(os.Stderr, "gen-faq-to input-path output-path")
+	os.Exit(1)
+}
+
+var anchor = regexp.MustCompile(`### <a name="(.*?)"></a> (.*)`)
+
+type tocentry struct {
+	anchor, title string
+}
+
+const (
+	startOfMappingTable = "<!-- BEGIN TOC -->"
+	endOfMappingTable   = "<!-- END TOC -->"
+)
+
+func spliceDocs(docpath string, docs []byte, outpath string) {
+	docbuf, err := ioutil.ReadFile(docpath)
```

Comment:
`ioutil` is deprecated since [Go 1.16](https://go.dev/doc/go1.16#ioutil). Let's use `os.ReadFile` instead.

Reply:
Not worth compromising compilation on older versions of Go for a deprecation.